src/mixins/common-mixin.ts

```diff
@@ -43,4 +43,20 @@ export default class CommonMixin extends Vue {
   isSame (objA: {}, objB: {}, prop: string = null): boolean {
     return isEqual({ ...this.omitProp(objA, [prop]) }, { ...this.omitProp(objB, [prop]) })
   }
+
+  /**
+   * Deconstructs and sorts an array of comments
```

Comment:
It doesn't deconstruct at all. This is deconstruction:
```
var [first, second, third] = ["Laide", "Gabriel", "Jets"]
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment